cypress/support/server.ts

```diff
@@ -2016,3 +2022,123 @@ export const mockDeleteFavorite = (shouldThrowError: boolean): void => {
     },
   ).as('unfavoriteItem');
 };
+
+// Intercept ShortLinks calls
+export const mockGetShortLinksItem = (
+  _shortLinks: ShortLink[],
+  shouldThrowError: boolean,
+): void => {
+  cy.intercept(
+    {
+      method: HttpMethod.GET,
+      url: new RegExp(`${API_HOST}/${buildGetShortLinksItemRoute(ID_FORMAT)}`),
+    },
+    ({ reply }) => {
+      if (shouldThrowError) {
+        return reply({ statusCode: StatusCodes.BAD_REQUEST });
+      }
+
+      return reply(_shortLinks);
+    },
+  ).as('getShortLinksItem');
+};
+
+export const mockCheckShortLink = (shouldThrowError: boolean): void => {
```

Comment:
What does "check" do? Is it the validity or the availability?

Reply:
It is the availability. The backend was updated to avoid confusing.